platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -120,7 +133,6 @@ public void init() throws Exception {
         return migrationWarnings;
     }
 
-    @Override
```

Comment:
Do we no longer need `@Override` here since we're just implementing an interface member? Just checking.